docs/HOOKS.md

```diff
@@ -715,6 +741,26 @@ This function is for rendering typing indicator for all participants of the conv
 -  `typing` lists participants who did not explicitly stopped typing. This list is a superset of `activeTyping`.
 -  `visible` indicates whether typing indicator should be shown in normal case. This is based on participants in `activeTyping` and their `role` (role not equal to `"user"`).
 
+## `useScrollTo`
+
+<!-- prettier-ignore-start -->
+```js
+useScrollTo(): (position: ScrollPosition, options: ScrollOptions) => void
+
+type ScrollOptions {
+  behavior: 'auto' | 'smooth'
+}
+
+type ScrollPosition {
+  scrollTop: number
+}
+```
+<!-- prettier-ignore-end -->
+
+This function will return a function that, when called, will scroll the transcript to the specific scroll position.
```

Comment:
```suggestion
This hook will return a function that, when called, will scroll the transcript to the specific scroll position.
```

Reply:
Looks like we should also update other hooks description. I will file a doc issue.